File: Graph/1791. Find Center of Star Graph.py
class Solution:
    def findCenter(self, edges: List[List[int]]) -> int:
        n = len(edges)
        ans = [0] * (n + 1)
        for e in edges:
            a, b = e[0] - 1, e[1] - 1
            ans[a] += 1
            ans[b] += 1
            if ans[a] == n:
                return a + 1
            if ans[b] == n:
                return b + 1

File: Graph/1791. Find Center of Star Graph.cpp
class Solution {
public:
    int findCenter(vector<vector<int>>& edges) {
        int n = size(edges);
        vector<int> ans(n + 1, 0);
        int a, b;
        for (auto e : edges){
            a = e[0] - 1;
            b = e[1] - 1;
            ans[a]++;
            ans[b]++;
            if (ans[a] == n)
                return a + 1;
            if (ans[b] == n)
                return b + 1;
        }
        return 0;
    }
};